### PR TITLE
Refactor repo2coocc entry point

### PR DESCRIPTION
**Pull request by zurk**
- Add min_docfreq parameter
- Add saving for document frequency model
- Add Extractors usage
- Delete outdated token_mapper transformer. We have Indexer now

Signed-off-by: Konstantin Slavnov konstantin@sourced.tech

### CODE REVIEW

File: sourced/ml/__main__.py [hunk 1]
```diff
@@ -99,6 +99,16 @@ def add_default_args(my_parser):
     repo2coocc_parser.add_argument(
         "-o", "--output", required=True,
         help="Path to the output file.")
+    repo2coocc_parser.add_argument(
```
Comment (vmarkovtsev) [inline sourced/ml/__main__.py]: Should be named `repos2coocc`

File: sourced/ml/__main__.py [hunk 2]
```diff
@@ -99,6 +99,16 @@ def add_default_args(my_parser):
...
+        "--split_stem", default=False, action="store_true",
```
Comment (vmarkovtsev) [inline sourced/ml/__main__.py]: No underscore is allowed in arguments

File: sourced/ml/cmd_entries/repos2coocc.py [hunk 3]
```diff
@@ -1,29 +1,42 @@
...
+        .execute()\
+        .filter(lambda x: x.value >= args.min_docfreq)
+    df = df_rdd.collectAsMap()
+    df_model = OrderedDocumentFrequencies() if args.ordered else DocumentFrequencies()
+    df_model.construct(df_transformer.ndocs, df)
+    log.info("Writing document frequency model to %s...", args.docfreq)
+    df_model.save(args.docfreq)
 
+    tokens = list(df.keys())
+    token2index = df_rdd.context.broadcast({token: i for i, token in enumerate(tokens)})
```
Comment (vmarkovtsev) [inline sourced/ml/cmd_entries/repos2coocc.py]: You do not need to convert to a list to enumerate

Comment (zurk) [reply sourced/ml/cmd_entries/repos2coocc.py]: I need both, `tokens ` as index2token and `token2index` as token2index,

Comment (vmarkovtsev) [reply sourced/ml/cmd_entries/repos2coocc.py]: ok

**Pull request merged**